sdk/storage/storage-blob/src/utils/tracing.ts

```diff
@@ -21,12 +21,17 @@ export function createSpan(
   };
 
   const span = tracer.startSpan(`Azure.Storage.Blob.${operationName}`, spanOptions);
+  span.setAttribute("az.namespace", "Microsoft.Storage");
```

Comment:
Just checking - we always intend to just use the most generic moniker possible? Ie, we wouldn't add in Microsoft.Storage.Blob?

Reply:
Per other comment, this is from the resource provider list: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers